### PR TITLE
[zh] update broken links of task/debug/debug-application-cluster

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

**Comment by howieyuen** [comment]
retitle [zh] update broken links of task/debug/debug-application-cluster

**Comment by howieyuen** [comment]
/retitle [zh] update broken links of task/debug/debug-application-cluster

**Comment by howieyuen** [comment]
Could you please check all these /debug/debug-xxx links in /zh ?
As I know:
`/task/debug/debug-application-cluster `
   -> `/task/debug/debug-application` or
   -> `/task/debug/debug-cluster`

**Comment by kinzhi** [comment]
> 
> Could you please check all these /debug/debug-xxx links in /zh ? As I know: `/task/debug/debug-application-cluster ` -> `/task/debug/debug-application` or -> `/task/debug/debug-cluster`

I have checked the required link and it is ok

**Comment by howieyuen** [comment]
thanks
/approve
/lgtm

File: content/zh/blog/_posts/2020-12-02-dockershim-faq.md [hunk 1]
```diff
@@ -267,13 +267,13 @@ runtime where possible.
...
 or nested inside a container when building images will no longer work. For the
-former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
+former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
 latter you can use newer container build options like [img], [buildah],
 [kaniko], or [buildkit-cli-for-kubectl] that don’t require Docker.
 -->
 另外还有一个需要关注的点，那就是当创建镜像时，系统维护或嵌入容器方面的任务将无法工作。
 对于前者，可以用 [`crictl`](https://github.com/kubernetes-sigs/cri-tools) 工具作为临时替代方案
-(参见 [从 docker 命令映射到 crictl](https://kubernetes.io/zh/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
+(参见 [从 docker 命令映射到 crictl](https://kubernetes.io/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-12-02-dockershim-faq.md:276]:
```suggestion
(参见[从 docker 命令映射到 crictl](/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
```

File: content/zh/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 2]
```diff
@@ -312,13 +312,13 @@ runtime where possible.
...
 or nested inside a container when building images will no longer work. For the
-former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
+former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
 latter you can use newer container build options like [img], [buildah],
 [kaniko], or [buildkit-cli-for-kubectl] that don’t require Docker.
 -->
 另外还有一个需要关注的点，那就是当创建镜像时，系统维护或嵌入容器方面的任务将无法工作。
 对于前者，可以用 [`crictl`](https://github.com/kubernetes-sigs/cri-tools) 工具作为临时替代方案
-(参阅[从 docker cli 到 crictl 的映射](/zh/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl))。
+(参阅[从 docker cli 到 crictl 的映射](/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-02-17-updated-dockershim-faq.md:321]:
```suggestion
(参阅[从 docker CLI 到 crictl 的映射](/zh/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))。
```

**Pull request merged**